packages/mask/src/components/DataSource/useActivatedUI.ts

```diff
@@ -64,6 +64,14 @@ export function useIsCurrentVisitingOwnerIdentity() {
     )
 }
 
+export function useIsMyIdentity(identity: IdentityResolved) {
```

Comment:
The `SocialIdentity`  defined a property `isOwner` for this.

Reply:
Only the result of `useCurrentVisitingSocialIdentity` provides `isOwner`.

BTW, I don't think it's good to add such a property to `SocialIdentity`, it depends on the context of `me`.

Reply:
Expect that the `useLastRecognizedIdentity` should always return `isOwn` as `true`.

Reply:
In ProfileCard, Identity could be anyone's.

Reply:
I made some refactoring https://github.com/DimensionDev/Maskbook/pull/7208. Check those new hooks:

- `useIsOwnerIdentity`
- `useSocialIdentity`